utils/cliutils/commandsflags.go

```diff
@@ -1418,6 +1431,35 @@ var flagsMap = map[string]cli.Flag{
 		Name:  Status,
 		Usage: "[Default: false] Set to true to show the status of the transfer-files command currently in progress.` `",
 	},
+
+	branch: cli.StringFlag{
+		Name:  branch,
+		Usage: "[Optional] Set branch name to filter default value is empty",
```

Comment:
Please make sure to stick to the conventions:
```suggestion
		Usage: "[Optional] Branch name to filter.` `",
```
Same for the other flags